protocol/common/message.go

```diff
@@ -703,12 +703,13 @@ func (m *Message) LoadImage() error {
 	return nil
 }
 
-func (m *Message) SetAlbumID(albumID string) error {
+func (m *Message) SetAlbumID(albumID string, ImagesCount uint32) error {
 	imageMessage := m.GetImage()
 	if imageMessage == nil {
 		return errors.New("Image is empty")
 	}
 	imageMessage.AlbumId = albumID
+	imageMessage.AlbumImagesCount = ImagesCount
```

Comment:
Here you have `ImageMessage` instance which doesn't have `AlbumImagesCount`.  Is it should be `m.AlbumImagesCount`?

Reply:
Shouldn't it be defaulted to `0` automatically ?